src/clr-angular/data/datagrid/render/header-renderer.ts

```diff
@@ -118,4 +124,12 @@ export class DatagridHeaderRenderer implements OnDestroy {
       this.autoSet = true;
     }
   }
+
+  private setVisibility(state: DatagridColumnState) {
```

Comment:
We should standardize the nomenclature around `hidden` or `visibility`. Since the feature is called hideable columns, I think hidden is more clear though technically we are setting the visibility property in the class.